pkg/server/dns_server.go

```diff
@@ -28,6 +28,7 @@ type DNSServer struct {
 	server        *dns.Server
 	customRecords *customDNSRecords
 	TxtRecord     string // used for ACME verification
+
```

Comment:
extra newline

Reply:
you can remove it

Reply:
The redundant blank lines have been removed. Please merge the main branches. Thank you very much

Reply:
I guess you should merge your pull request by yourself after it is approved